### ISSUE TITLE
Index was outside the bounds of the array. at getIcon() in Wox.Plugin.ControlPanel\ControlPanelList.cs:line 222

### ISSUE THREAD

**Issue by meixger** [Wox-launcher/Wox#846]
1. version 1.3.183
2. error happens immediately on start
3. Windows 10 Pro 1511.10586.420
4. Exception Info

```
Wox version: 1.3.183
OS Version: Microsoft Windows NT 10.0.10586.0
Date: 07/11/2016 12:19:09
Exception:
Wox.Plugin.ControlPanel
System.IndexOutOfRangeException
Index was outside the bounds of the array.
   at Wox.Plugin.ControlPanel.ControlPanelList.getIcon(RegistryKey currentKey, Int32 iconSize) in C:\projects\wox\Plugins\Wox.Plugin.ControlPanel\ControlPanelList.cs:line 222
   at Wox.Plugin.ControlPanel.ControlPanelList.Create(UInt32 iconSize) in C:\projects\wox\Plugins\Wox.Plugin.ControlPanel\ControlPanelList.cs:line 77
   at Wox.Plugin.ControlPanel.Main.Init(PluginInitContext context) in C:\projects\wox\Plugins\Wox.Plugin.ControlPanel\Main.cs:line 20
   at Wox.Infrastructure.Stopwatch.Normal(String name, Action action) in C:\projects\wox\Wox.Infrastructure\Stopwatch.cs:line 28
   at Wox.Core.Plugin.PluginManager.<>c.<InitializePlugins>b__19_0(PluginPair pair) in C:\projects\wox\Wox.Core\Plugin\PluginManager.cs:line 96
   at System.Threading.Tasks.Parallel.<>c__DisplayClass17_0`1.<ForWorker>b__1()
   at System.Threading.Tasks.Task.InnerInvokeWithArg(Task childTask)
   at System.Threading.Tasks.Task.<>c__DisplayClass176_0.<ExecuteSelfReplicating>b__0(Object )
```
1. Log file [2016-07-11.txt](https://github.com/Wox-launcher/Wox/files/356866/2016-07-11.txt)

**Comment by meixger** [comment]
This happens when the registry key `HKCR\CLSID\{xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx}\DefaultIcon` has a empty `(Default)` value.

[ControlPanelList.cs#L233](https://github.com/Wox-launcher/Wox/blob/master/Plugins/Wox.Plugin.ControlPanel/ControlPanelList.cs#L233) `if (iconString[0][0] == '@') // IndexOutOfRangeException`

**Comment by meixger** [comment]
fixed by #917
